File: VisualBasicAssignment/screen/sai_keat/widget/order/StaffOrder.cs
using System;
using System.Windows.Forms;

namespace VisualBasicAssignment
{
    public partial class StaffOrder
    {
        public StaffOrder()
        {
            InitializeComponent();
            _lnkPaymentReport.Name = "lnkPaymentReport";
            _linkSalesReport.Name = "linkSalesReport";
            _lnkNewOrder.Name = "lnkNewOrder";
        }

        public void switchPanel2(Control panel)
        {
            Panel1.Controls.Clear();
            panel.Show();
            panel.Visible = true;
            Panel1.Controls.Add(panel);
        }

        private void lnkNewOrder_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            switchPanel2(new NewOder());
        }

        private void linkSalesReport_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            switchPanel2(new SalesReport());
        }

        private void lnkPaymentReport_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            switchPanel2(new PaymnetReport());
        }

        private void StaffOrder_Load(object sender, EventArgs e)
        {
            switchPanel2(new NewOder());
        }
    }
}

File: VisualBasicAssignment/screen/sai_keat/widget/order/StaffOrder.vb
Public Class StaffOrder

    Sub switchPanel2(ByVal panel As Control)
        Panel1.Controls.Clear()
        panel.Show()
        panel.Visible = True
        Panel1.Controls.Add(panel)
    End Sub

    Private Sub lnkNewOrder_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles lnkNewOrder.LinkClicked
        switchPanel2(New NewOder())
    End Sub

    Private Sub linkSalesReport_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles linkSalesReport.LinkClicked
        switchPanel2(New SalesReport())
    End Sub

    Private Sub lnkPaymentReport_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles lnkPaymentReport.LinkClicked
        switchPanel2(New PaymnetReport())
    End Sub

    Private Sub StaffOrder_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        switchPanel2(New NewOder())
    End Sub
End Class
